syntax = "proto3"; // Set to proto2 if you are using Protocol Buffer Version 2

package arithmetic_proto;

message SumRequest
{
    float number_1 = 1;
    float number_2 = 2;
}

message SumResponse
{
    float answer = 1;
}

service SumService
{
    rpc Sum(SumRequest) returns (SumResponse) {};
}
